ts/Extensions/ExportData/ExportDataOptions.d.ts

```diff
@@ -31,6 +31,7 @@ declare module '../../Core/Options' {
         exportData?: ExportDataLangOptions;
         viewData?: string;
         hideData?: string;
+        exportInProgress?: string;
```

Comment:
Possibly not needed due to `lang` object.

Reply:
It's not necessarily defined in the sense that it requires the export-data module to be loaded for the end user.